src/python/turicreate/toolkits/activity_classifier/_tf_model_architecture.py

```diff
@@ -21,14 +21,15 @@
 
 class ActivityTensorFlowModel(TensorFlowModel):
 
-    def __init__(self, net_params, batch_size, num_features, num_classes, prediction_window, seq_len):
+    def __init__(self, net_params, batch_size, num_features, num_classes, prediction_window, seq_len, random_seed):
 
+        _utils.suppress_tensorflow_warnings()
         self.gpu_policy = _utils.TensorFlowGPUPolicy()
         self.gpu_policy.start()
 
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
-
+        seed = random_seed if random_seed!=-1 else None
```

Comment:
Why isn't `-1` a valid value for the seed?

Reply:
I should try to send random seed as FLEX_UNDEFINED at inference time.